.github/workflows/build-pr.yml

```diff
@@ -20,3 +20,6 @@ jobs:
         STRUCTURIZR_WORKSPACE_ID: ${{secrets.STRUCTURIZR_WORKSPACE_ID}}
         STRUCTURIZR_API_KEY: ${{secrets.STRUCTURIZR_API_KEY}}
         STRUCTURIZR_API_SECRET: ${{secrets.STRUCTURIZR_API_SECRET}}
+    - name: sample build
+      run: cd sample && npm i && npm run build
```

Comment:
I don't think this will help in verifying that the current version that's in git (e.g. the one in a PR) is working correctly and not break the sample, since the sample projects package.json references a specific version from the npm repository.